src/test/java/com/VetApp/PurrgentCare/services/PetServiceImplementationTest.java

```diff
@@ -63,11 +63,19 @@ public void getPet_whenExist_returnOnePet() {
     @Test
     public void getPet_whenNotExist_returnEmptyPetResponse() {
         // given
-        final var fakePetId = fakeDataGenerator.generateRandomInteger();
-        final var fakePetResponse = new PetResponse();
+        final var fakePet = fakeDataGenerator.generateFakePet();
+        final var fakePetId = fakePet.getId();
+        final var fakePetResponse = fakeDataGenerator.generateFakePetResponse();
+        fakePetResponse.setId(null);
+        fakePetResponse.setName(null);
+        fakePetResponse.setType(null);
+        fakePetResponse.setAge(null);
+        fakePetResponse.setGender(null);
```

Comment:
Similar to the Account comment, is there a way we could either make a method to generate a fake pet response or initialize this in the test to be an empty response?